framework/include/wifi_manager/wifi_manager.h

```diff
@@ -283,7 +305,7 @@ wifi_manager_result_e wifi_manager_get_info(wifi_manager_info_s *info);
  * @brief Connect to an access point.
  * @details @b #include <wifi_manager/wifi_manager.h>
  * @param[in] config ssid, passphrase, authentication type, and cryto type of the access point which the wi-fi interface connect to.
```

Comment:
Spell mistakes needs to be corrected
cryto => crypto

Reply:
Modification completed. Thanks.